code/__DEFINES/layers.dm

```diff
@@ -146,3 +146,5 @@
 
 #define CINEMATIC_LAYER 21
 
+#define SPLASHSCREEN_LAYER 22
```

Comment:
`SPLASHSCREEN_LAYER` is /tg name, `CINEMATIC_LAYER` is our codebase name.
Fuse them (your choice) and add `CINEMATIC/SPLASHSCREEN_PLANE`.